web/themes/custom/cj_material/sass/styles.scss

```diff
@@ -1375,11 +1376,282 @@ article footer {
   font-style: italic;
 }
 
+/* BP - Job Post Page */
+
 .form-item label {
-  font-size: 14px;
+  font-size: 20px;
   font-weight: bold;
+  font-family: Montserrat;
+  line-height: 2em;
 }
 
+.form-item input[type=text], .form-item input[type=email], .form-item input[type=password], .form-item input[type=search], .form-item input[type=tel], .form-item textarea, .form-item select {
+    display: block;
+    width: 100%;
+    max-width: 1200px;
+    border: 1px solid rgba(0, 0, 0, 0.12);
+    padding: 8px 0;
+}
+
+input#edit-label-0-value {
+    background-color: #f6f4f4;
+    text-indent: 10px;
+}
+
+textarea#edit-description-summary-0-value {
+    background-color: #f6f4f4;
+    text-indent: 10px;
+}
+
+form#job-role-post-form {
+    padding-left: 20px;
+    padding-right: 20px;
+}
+
+/* BP Post a Job 21/1/2020 */
+
+summary {
+    border: 2px solid;
+    border-color: #ff8f48;
+}
+
+.divider-top:before {
+    background: #FF904B;
+    content: "";
+    height: 6px;
+    position: absolute;
+    left: -40px;
+    top: 0px;
+    width: 1200px;
+}
+
+.form-item input[type=text]:focus, .form-item input[type=text].error:focus, .form-item input[type=email]:focus, .form-item input[type=email].error:focus, .form-item input[type=password]:focus, .form-item input[type=password].error:focus, .form-item input[type=search]:focus, .form-item input[type=search].error:focus, .form-item input[type=tel]:focus, .form-item input[type=tel].error:focus, .form-item textarea:focus, .form-item textarea.error:focus, .form-item select:focus, .form-item select.error:focus {
+    border: 2px solid #FF904B;
+    padding-bottom: 7px;
+}
+
+div#edit-description-wrapper {
+    padding-bottom: 20px;
+}
+
+div.details-wrapper {
+    padding-left: 0px;
+    padding-right: 0px;
+}
+
+select#edit-salary-0-compensation {
+    text-indent: 10px;
+}
+
+span.field-prefix {
+    padding-left: 10px;
+    padding-right: 10px;
+    font-size: 20px;
+}
+
+input#edit-salary-0-from {
+    border-color: rgba(0,0,0,0.12);
+}
+
+input#edit-salary-0-from {
+    background-color: #f6f4f4;
+    border: 1px solid;
+}
+
+input#edit-salary-0-to {
+    border-color: rgba(0,0,0,0.12);
+}
+
+input#edit-salary-0-to {
+    background-color: #f6f4f4;
+    border: 1px solid;
+}
+
+select#edit-description-0-format--2 {
+    text-indent: 10px;
+    background-color: #f6f4f4;
+}
+
+select#edit-industry {
+    text-indent: 10px;
+    background-color: #f6f4f4;
+}
+
+select#edit-hours {
+    text-indent:  10px;
+    background-color: #f6f4f4;
+}
+
+input.select2-search__field {
+    text-indent: 10px;
+    background-color: #f6f4f4;
+}
+
+select#edit-location-0-type {
+    text-indent: 10px;
+    background-color: #f6f4f4;
+}
+
+select#edit-location-0-address-country-code--2 {
+    text-indent: 10px;
+    background-color: #f6f4f4;
+}
+
+select#edit-contact-address-0-address-country-code--2 {
+    text-indent: 10px;
+    background-color: #f6f4f4;
+}
```

Comment:
You can group these together, I don't know what the class is off the top of my head

```sass
.field--type-address {
  select.country {
    text-indent: 10px;
    background-color: #F6F4F4;
  }
}
```

Reply:
Also noticing that combination of text indent and background colour are VERY common, lets try and design our CSS so we only write it once.

Reply:
```sass
select {
  text-indent: 10px;
  background-color: #F6F4F4;
}
```